src/sweeper/__main__.py

```diff
@@ -98,11 +121,25 @@ def run_tool(tool):
 
             continue
 
-        print('missing table, executing over workspace')
-
         #: get feature class names once
         if len(feature_class_names) == 0:
-            feature_class_names = workspace_info.get_featureclasses(tool.workspace)
+            if change_detect:
+                print('Getting table names from change detection table')
```

Comment:
**chore**: remove all print statements and replace with logs or delete them if they aren't log worthy.

Reply:
We originally included print statements with the idea that Sweeper could be run by anyone (and used by other offices) to check a single feature class for issues from the command line.  Should we instead direct all output to a single log?  Would that force the user to specify a log file?  Right now, each feature class and test has it's own report file if the `--save-report` flag is used.

Reply:
Python logging can have multiple handlers - console, file, whatever. I would suggest that the standard out (console) handler is always added. If the `--save-report` flag is added then add the file handler.

Reply:
I created an issue for this.